L/LibBirch/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "LibBirch"
+version = v"1.0.0"
```

Comment:
Birch does not use semver and instead "The master branch is considered stable". There exist v0 and v1 tags - what is the correct way to deal with this in Yggdrasil?

Reply:
Maintain a list of semver  (versions that you get to make up) to git hashes...

Reply:
Also see my comments [here](https://github.com/JuliaPackaging/Yggdrasil/pull/2156/files#r528224861).

Reply:
Yes, I think it's probably best to just the same versioning scheme as upstream here.

Reply:
```suggestion
# Birch's version is `(git describe --long || echo) | sed -E 's/v([0-9]+)-([0-9]+)-g[0-9a-f]+/\1.\2/'`
# (see https://github.com/JuliaPackaging/Yggdrasil/pull/2156#discussion_r528224861)
version = v"1.75"
```